### PR TITLE
Clean up backend and frontend templating

### PR DESCRIPTION
**Pull request by carakas**
## Type

- Enhancement
## Pull request description

Clean up the backend and frontend templating engine mess

### CODE REVIEW

File: src/Frontend/Core/Engine/TwigTemplate.php [hunk 1]
```diff
@@ -26,31 +22,17 @@ class TwigTemplate extends BaseTwigTemplate
...
+        Environment $environment,
         TemplateNameParserInterface $parser,
         FileLocatorInterface $locator
     ) {
-        $container = Model::getContainer();
-        $this->forkSettings = $container->get('fork.settings');
         $this->language = Locale::frontendLanguage();
 
-        parent::__construct(clone $environment, $parser, clone $locator);
+        parent::__construct(clone $environment, $parser, clone $locator, 'Frontend');
```
Comment (StijnVrolijk) [inline src/Frontend/Core/Engine/TwigTemplate.php]: Can't `'Frontend'` be replaced by `APPLICATION`?

File: src/Common/Core/Twig/BaseTwigTemplate.php [hunk 2]
```diff
@@ -9,10 +9,10 @@
 use SpoonForm;
 use Symfony\Bridge\Twig\Form\TwigRendererEngine;
 use Symfony\Bundle\TwigBundle\TwigEngine;
+use Symfony\Component\Stopwatch\Stopwatch;
```
Comment (StijnVrolijk) [inline src/Common/Core/Twig/BaseTwigTemplate.php]: Where did this come from?

**Review by StijnVrolijk — CHANGES_REQUESTED** [review]

File: src/Common/Core/Twig/BaseTwigTemplate.php (hunk 2 above)
Comment (carakas) [reply src/Common/Core/Twig/BaseTwigTemplate.php]: From debugging it :D should be removed

File: src/Frontend/Core/Engine/TwigTemplate.php (hunk 1 above)
Comment (carakas) [reply src/Frontend/Core/Engine/TwigTemplate.php]: No, cause then you won't be able to use it correctly in a console command

File: src/Common/Core/Twig/BaseTwigTemplate.php (hunk 2 above)
Comment (StijnVrolijk) [reply src/Common/Core/Twig/BaseTwigTemplate.php]: Very well, remove it and i'll approve this PR

**Pull request closed without merging**